packages/generator-open-wc/generators/starter-app/templates/static/.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
```

Comment:
@daKmoR I was hoping this would overwrite the `.eslintrc.js` that gets made by the generator, but it doesnt. How should I go about this?

Same for the `webpack.config.js`

Reply:
for now just use `conflicts()` instead of `writing()` in you index.js of the generator (e.g. a task that gets executed later)

we may want to test if moving all of our tasks (in every generator) to default might solve this as well... but that is for later

Reply:
I adjusted all generators to use proper init and use default to do the copying... so we can us writing in this generator